spec/support/approximating_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ApproximatingHelper
+  def range_approximating(size, vary_left: nil, vary_right: nil)
```

Comment:
we only use this method in the one file, so I'd have just defined it inside there, but this is fine too

Also it looks like `vary_left` is never used so in the spirit of YAGNI, I would remove it